jscomp/test/alias_default_value_test.res

```diff
@@ -0,0 +1,18 @@
+@@bs.config({
+  flags: ["-bs-jsx", "4"],
+})
+
+module C0 = {
+  let a = 1
```

Comment:
Can you remove this line. as it's not needed.

Reply:
Got it https://github.com/rescript-lang/rescript-compiler/pull/5989/commits/6c0a0bb4070f5e94087cc96552b5744ddecc9ddd